### PR TITLE
Add resource_name to TX definition

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #11839 (Harmonize naming translations resources)

Add a `resorce_name` property to each TX definition (`packages/xxx/.tx/config`)

`activities` --> `web-activities` (`<repo-name>-<resource-name>`

Now, we can destinguish by the name in TX where the source is located:
`web-yyy`
`web-extension-yyy`
...

### CODE REVIEW

File: packages/web-runtime/l10n/.tx/config [hunk 1]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:core]
+[o:owncloud-org:p:owncloud-web:r:web-core]
```
Comment (kulmann) [inline packages/web-runtime/l10n/.tx/config:4]: Please name this "web-runtime". Don't know why this was core in the past, but the package name is runtime and should be reflected in the resource name.

File: changelog/unreleased/enhancement-resource-name
Comment (kulmann) [inline changelog/unreleased/enhancement-resource-name:1]: Please remove the changelog item again. The change is not related to the software so it would be irritating to have it.

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/web-runtime/l10n/.tx/config (hunk 1 above)
Comment (mmattel) [reply packages/web-runtime/l10n/.tx/config]: done

File: changelog/unreleased/enhancement-resource-name
Comment (mmattel) [reply changelog/unreleased/enhancement-resource-name:1]: done

File: packages/design-system/l10n/.tx/config [hunk 2]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:design-system]
+[o:owncloud-org:p:owncloud-web:r:web-design-system]
```
Comment (kulmann) [inline packages/design-system/l10n/.tx/config:4]: oh, just seeing this: you need to leave line 4 in all of the packages untouched, otherwise this will recreate all resources with new slugs on transifex. You must only add the `resource_name` property, nothing else.

File: packages/web-runtime/l10n/.tx/config (hunk 1 above)
Comment (kulmann) [reply packages/web-runtime/l10n/.tx/config]: unresolving, this needs to stay `core`, only the resource_name should be `web-runtime`. Sorry for the confusion.

File: packages/design-system/l10n/.tx/config (hunk 2 above)
Comment (mmattel) [inline packages/design-system/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:design-system]
```

File: packages/web-app-activities/l10n/.tx/config [hunk 3]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:activities]
+[o:owncloud-org:p:owncloud-web:r:web-activities]
```
Comment (mmattel) [inline packages/web-app-activities/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:activities]
```

File: packages/web-app-app-store/l10n/.tx/config [hunk 4]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:app-store]
+[o:owncloud-org:p:owncloud-web:r:web-app-store]
```
Comment (mmattel) [inline packages/web-app-app-store/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:app-store]
```

File: packages/web-app-epub-reader/l10n/.tx/config [hunk 5]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:epub-reader]
+[o:owncloud-org:p:owncloud-web:r:web-epub-reader]
```
Comment (mmattel) [inline packages/web-app-epub-reader/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:epub-reader]
```

File: packages/web-app-external/l10n/.tx/config [hunk 6]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:external]
+[o:owncloud-org:p:owncloud-web:r:web-external]
```
Comment (mmattel) [inline packages/web-app-external/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:external]
```

File: packages/web-app-files/l10n/.tx/config [hunk 7]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:files]
+[o:owncloud-org:p:owncloud-web:r:web-files]
```
Comment (mmattel) [inline packages/web-app-files/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:files]
```

File: packages/web-app-ocm/l10n/.tx/config [hunk 8]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:ocm]
+[o:owncloud-org:p:owncloud-web:r:web-ocm]
```
Comment (mmattel) [inline packages/web-app-ocm/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:ocm]
```

File: packages/web-app-pdf-viewer/l10n/.tx/config [hunk 9]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:pdf-viewer]
+[o:owncloud-org:p:owncloud-web:r:web-pdf-viewer]
```
Comment (mmattel) [inline packages/web-app-pdf-viewer/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:pdf-viewer]
```

File: packages/web-app-preview/l10n/.tx/config [hunk 10]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:preview]
+[o:owncloud-org:p:owncloud-web:r:web-preview]
```
Comment (mmattel) [inline packages/web-app-preview/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:preview]
```

File: packages/web-app-search/l10n/.tx/config [hunk 11]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:search]
+[o:owncloud-org:p:owncloud-web:r:web-search]
```
Comment (mmattel) [inline packages/web-app-search/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:search]
```

File: packages/web-app-text-editor/l10n/.tx/config [hunk 12]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:text-editor]
+[o:owncloud-org:p:owncloud-web:r:web-text-editor]
```
Comment (mmattel) [inline packages/web-app-text-editor/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:text-editor]
```

File: packages/web-app-webfinger/l10n/.tx/config [hunk 13]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:webfinger]
+[o:owncloud-org:p:owncloud-web:r:web-webfinger]
```
Comment (mmattel) [inline packages/web-app-webfinger/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:webfinger]
```

File: packages/web-client/l10n/.tx/config [hunk 14]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:client]
+[o:owncloud-org:p:owncloud-web:r:web-client]
```
Comment (mmattel) [inline packages/web-client/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:client]
```

File: packages/web-pkg/l10n/.tx/config [hunk 15]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:pkg]
+[o:owncloud-org:p:owncloud-web:r:web-pkg]
```
Comment (mmattel) [inline packages/web-pkg/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:pkg]
```

File: packages/web-runtime/l10n/.tx/config [hunk 16]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:core]
+[o:owncloud-org:p:owncloud-web:r:web-runtime]
```
Comment (mmattel) [inline packages/web-runtime/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:runtime]
```

File: packages/web-runtime/l10n/.tx/config [hunk 17]
```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:core]
+[o:owncloud-org:p:owncloud-web:r:runtime]
```
Comment (mmattel) [inline packages/web-runtime/l10n/.tx/config:4]:
```suggestion
[o:owncloud-org:p:owncloud-web:r:core]
```

**Comment by mmattel** [comment]
@kulmann we now only have added `resource_name` in this PR

**Comment by mmattel** [comment]
Are CI things related to this PR, it does not change code? Was green before...

**Comment by kulmann** [comment]
> Are CI things related to this PR, it does not change code? Was green before... Rebasing

Seems to be unrelated

**Review by kulmann — APPROVED** [review]
Thank you for aligning this 👍
